spec/imsc-hrm.html

```diff
@@ -473,11 +473,11 @@ <h2>Architecture</h2>
 
       <p class="note">The HRM permits a maximum rate of 12 <a>Intermediate Synchronic Documents</a> per second. This is
       ultimately limited by the BDraw parameter and is intended to capture processing and presentation overhead. When
-      converting a [[CEA-608]] signal to IMSC, it is therefore impossible to create a separate <a>IMSC Document
-      Instance</a> for every [[CEA-608]] packet, which are sampled at approximately 59.94 Hz. It is instead preferable
-      to coalesce sequences of [[CEA-608]] packets into complete lines and paragraphs before creating a <a>IMSC Document
-      Instance</a>, and let the <a>presentation processor</a> perform any desired animation, e.g., typewriter
-      effect.</p>
+      converting a [[CEA-608]] signal to IMSC, it is therefore impossible to create <a>IMSC Document Instances</a> that
+      generate an <a>Intermediate Synchronic Documents</a> for every [[CEA-608]] packet, which are sampled at
```

Comment:
sorry, probably my typo in the suggestion before.

```suggestion
      generate an <a>Intermediate Synchronic Document</a> for every [[CEA-608]] packet, which are sampled at
```